model.go

```diff
@@ -237,3 +237,12 @@ type Language struct {
 	Status string `json:"status"`
 	Name   string `json:"name"`
 }
+
+// FollowerIDs represents the follower ids obejct received from Twitter followers/ids
+type FollowerIDs struct {
+	IDs               []int  `json:"ids"`
```

Comment:
Should be `int64` (the type for all Twitter numerical IDs). Does this endpoint return an `id_strs` field as well?